shunting-yard-common/src/main/java/com/hotels/shunting/yard/common/emitter/AbstractMetaStoreEventListener.java

```diff
@@ -81,44 +83,72 @@ private Message withPayload(SerializableListenerEvent event) throws MetaExceptio
 
   @Override
   public void onCreateTable(CreateTableEvent tableEvent) throws MetaException {
-    LOG.info("Create table event received");
-    executorService.submit(message(withPayload(serializableListenerEventFactory.create(tableEvent))));
+    log.info("Create table event received");
+    try {
```

Comment:
An Aspect could be a slightly nicer way to do this but perhaps not so relevant for only 7 methods, also given the issues we had with getting Aspects to work last time we tried using them.